interpreter/cling/lib/Interpreter/InterpreterCallbacks.cpp

```diff
@@ -239,6 +241,27 @@ namespace cling {
     return false;
   }
 
+  void InterpreterCallbacks::CreateCodeCompleteConsumer(Interpreter* child) const {
+    ClingCodeCompleteConsumer* consumer = new ClingCodeCompleteConsumer(
+                m_Interpreter->getCI()->getFrontendOpts().CodeCompleteOpts,
+                llvm::outs());
```

Comment:
`outs()` and `stdout` (the latter is used by ROOT) interfere, or at least did so in the past. Can we use `llvm::errs()` here?